src/r5py/r5/transport_network.py

```diff
@@ -62,6 +64,47 @@ def __init__(self, osm_pbf, gtfs=[], build_config={}):
         )
         self._transport_network.transitLayer.buildDistanceTables(None)
 
+    @classmethod
+    def from_directory(cls, path):
+        """
+        Load a transport network, find input data in `path`.
+
+        This mimicks r5r’s behaviour to accept a directory path
+        as the only input to `setup_r5()`.
+
+        If more than one OpenStreetMap extract (`.osm.pbf`) is
+        found in `path`, the (alphabetically) first is used.
+        In case *no* OpenStreetMap extract is found, a `FileNotFound`
+        exception is raised. Any and all GTFS data files are used.
+
+        Arguments:
+        ----------
+        path : str
+            directory path in which to search for GTFS and .osm.pbf files
+
+        Returns:
+        --------
+        TransportNetwork
+            A fully initialised r5py.TransportNetwork
+        """
+        path = pathlib.Path(path)
+        try:
+            osm_pbf = sorted(path.glob("*.osm.pbf"))[0]
```

Comment:
We don't need to do anything for this now, but perhaps would be good to add a TODO comment here, so that we deal with this somehow in the future. Now we just assume to use the first osm dump that is found, but if there are more of them, I think we should at least warn the user, perhaps even raise an Exception because an incorrect OSM pbf matched with the transit network can cause unexpected behavarior.